local-cli/runIOS/runIOS.js

```diff
@@ -52,10 +52,10 @@ function _runIOS(argv, config, resolve, reject) {
     throw new Error(`Cound't find ${args.simulator} simulator`);
   }
 
-  const simulatorFullName = `${selectedSimulator.name} (${selectedSimulator.version})`;
+  const simulatorFullName = `${selectedSimulator.name} (${selectedSimulator.version}) [${selectedSimulator.udid}]`;
```

Comment:
Can we then remove the udid from fullName? I think that's more than we need to print on line 56